src/mam4xx/convproc.hpp

```diff
@@ -2844,6 +2856,338 @@ void ma_convproc_dp_intr(
+  // Does trace gases when convproc_do_gas is .true.
+  //
+  // Does deep and shallow convection
+  // Uses mass fluxes, cloud water, precip production from the
+  //    convective cloud routines
+  //
+  // Author: R. Easter
+  //
+  //-----------------------------------------------------------------------
+
+  // clang-format off
+  // Arguments
+  /*  
+  in    :: dpdry[nlev]           ! layer delta-p-dry [mb]
+  in    :: pdel[nlev]            ! layer delta-p [mb]
+  in    :: temperature[nlev]     ! Temperature [K]
+  in    :: pmid[nlev]            ! Pressure at model levels [Pa]
+
+  in    :: dt                        ! 2 delta t [s]
+                                ! from input this is just model time step, 
+                                ! not sure why it is "2" delta t. Shuaiqi Tang 2022
+  in    :: dp_frac[nlev]       ! Deep conv cloud frac [fraction]
+  in    :: icwmrdp[nlev]         ! Deep conv cloud condensate (in cloud) [kg/kg]
+  in    :: rprddp[nlev]        ! Deep conv precip production (grid avg) [kg/kg/s]
+  in    :: evapcdp[nlev]       ! Deep conv precip evaporation (grid avg) [kg/kg/s]
+  in    :: sh_frac[nlev]       ! Shal conv cloud frac [fraction]
+  in    :: icwmrsh[nlev]       ! Shal conv cloud condensate (in cloud) [kg/kg]
+  in    :: rprdsh[nlev]        ! Shal conv precip production (grid avg) [kg/kg/s]
+  in    :: evapcsh[nlev]       ! Shal conv precip evaporation (grid avg) [kg/kg/s]
+  in    :: dlf[nlev]           ! Tot  conv cldwtr detrainment (grid avg) [kg/kg/s]
+  in    :: dlfsh[nlev]         ! Shal conv cldwtr detrainment (grid avg) [kg/kg/s]
+  in    :: sh_e_ed_ratio[nlev] ! shallow conv [ent/(ent+det)] ratio [fraction]
+  inout :: aerdepwetis[pcnst]  ! aerosol wet deposition (interstitial) [kg/m2/s]
+                       ! eu, ed, du are "d(massflux)/dp" and are all positive
+  in    :: eu[nlev]    ! Mass entrain rate into updraft [1/s]
+  in    :: ed[nlev]    ! Mass entrain rate into downdraft [1/s]
+  in    :: du[nlev]    ! Mass detrain rate from updraft [1/s]
+  in    :: dp[nlev]    ! Delta pressure between interfaces [mb]
+  in    :: ktop           ! Index of cloud top (updraft top) for each column in w grid
+  in    :: kbot         ! Index of cloud base (level of maximum moist static energy) for each column in w grid
+  in    :: species_class[ConvProc::gas_pcnst]  ! species index defined as
+          spec_class::undefined  = 0
+          spec_class::cldphysics = 1
+          spec_class::aerosol    = 2
+          spec_class::gas        = 3
+          spec_class::other      = 4
+  
+  in  mmtoo_prevap_resConvProc::gas_pcnstusp[ConvProc::gas_pcnst]
+        pointers for resuspension mmtoo_prevap_resusp values are
+           >=0 for aerosol mass species with    coarse mode counterpart
+           -2 for aerosol mass species WITHOUT coarse mode counterpart
+           -3 for aerosol number species
+           -1 for other species
+  */ 
+  // clang-format off
```

Comment:
Should this be `clang-format on`?

Reply:
Oh, that explains why clang-format seemed to stop working.  I had turned it off when I  meant to turn it back on.